Overview.md

```diff
@@ -29,23 +29,79 @@ page_nav:
 
-<div class="example">
-    <p>insert HTML to render it within this example class</p>
-</div>
+Agenda connects to external scheduling applications like Microsoft Outlook Calendar, Google Calendar, or Zoom and merges their independent meeting data into one place. By consolidating team member availability into one simplified workspace, you can quickly schedule meetings that work well for everyone. Agenda eliminates the cumbersome process of comparing multiple calendars for team members across various applications.
 
-<br>
-Content should be in Markdown format and you shouldn't remove `.yml` comments on top of the `.md` files, errors may occur if you remove them.
+With Agenda, you can:
 
-<div class="callout callout--info">
-    <p><strong>Lorem ipsum dolor sit amet!</strong> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
-    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
-</div>
+- Quickly [view and schedule meetings](insert link to Schedule a Meeting tutorial).
+- [Create and manage teams](insert link to Create a Team tutorial) that suit your organizational needs.
+- [Import calendars and meeting rooms](insert link to Import a Meeting tutorial) from external applications.
+- Connect to your Employee Management System for quick [team onboarding](insert link to Import a Team tutorial).
 
-| First Header  | Second Header |
-| ------------- | ------------- |
-| Content Cell  | Content Cell  |
-| Content Cell  | Content Cell  |
+**add appropriate links to the above list items for quick access to their associated docs or headings in this doc**
+
+Not sure where to begin? See the [getting started guide](insert link to Quick Start Guide) to learn how to import your first calendar.
+  
+## How Agenda works
+
+Agenda connects to external calendars and meeting platforms to merge user meeting details into one simplified view, providing a more comprehensive picture of team availability and simplifying the scheduling workflow. With everyone's meeting details available within one space, you can quickly create teams and schedule meetings that work well for all.
+
+You can manage Agenda's capabilities through the simplified [user interface](UI.md) or integrate Agenda with your external applications using the [RESTful API](API.md).
+
+### Quick scheduling
+
+Agenda's home view displays your upcoming meetings and provides quick access to the [scheduling tool](insert link to the Schedule a Meeting tutorial) in one simplified user interface. 
+
+![](images/overview-home-view.png)
+
+From here, you can select the plus icon to start scheduling a new meeting. Agenda's scheduling workflow walks you through all of the necessary steps to set up a meeting with your team members and suggests recommended times based on everyone's availability.
```

Comment:
Re: "the plus icon", it's probably better to refer to it as the action the button represents rather than the symbol, something like "the add icon".

If possible, it would be awesome to include a small inline icon to include in brackets next to that for clarity, e.g. "...select the add icon (![add-icon](https://user-images.githubusercontent.com/44183179/197899158-824195ba-3fda-4694-80b5-16ed96bc3433.png)) to start...". (I went ahead and made that one from a screenshot in XD, feel free to use)